PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+Issue: (link to applicable gitHub Issue)
+Linked PRs: (links to corresponding PRs, optional)
+
+**Reviewers:** (List the people who you like to have this PR to be reviewed)
```

Comment:
I think this should be omitted. Github has an "assignee" field built in that we should try to use.